nlu/requirements.txt

```diff
@@ -1,9 +1,37 @@
-tensorflow==2.13.*
-transformers==4.30.2
-keras==2.13.*
-numpy==1.24.*
-scikit_learn==1.2.2
-fastapi==0.100.0
-uvicorn[standard]==0.23.1
-autopep8==2.0.2
-h5py --only-binary=h5py
+absl-py==2.1.0
```

Comment:
Can we keep the requirement.txt as is if we didn't install any extra packages ?

Reply:
Addressed. Only extra package needed is the focal_loss package